src/control/IntersectionReferenceSystem.ts

```diff
@@ -64,12 +67,18 @@ export class IntersectionReferenceSystem {
       curve: new CurveInterpolator(path),
       trajectory: new CurveInterpolator(
         path.map((d: number[]) => [d[0], d[1]]),
-        { tension, arcDivisions },
+        { tension: TENSION, arcDivisions: ARC_DIVISIONS },
       ),
-      curtain: new CurveInterpolator(this.projectedPath, { tension, arcDivisions }),
+      curtain: new CurveInterpolator(this.projectedPath, { tension: TENSION, arcDivisions: ARC_DIVISIONS }),
     };
 
-    this.endVector = IntersectionReferenceSystem.getDirectionVector(this.interpolators.trajectory, 1 - thresholdDirectionDist, 1);
+    if (trajectoryAngle) {
+      const angleInRad = (trajectoryAngle * Math.PI) / DEG_180;
```

Comment:
Use videx math radians function to convert degrees to rad

Reply:
done